pkg/server/http_handler_test.go

```diff
@@ -75,7 +87,7 @@ type mockStore struct {
 
 func (m *mockStore) CreateCustomer(name string) (*model.Customer, error) {
 	if m.createCustomerFunc != nil {
-		m.createCustomerFunc(name)
+		_, _ = m.createCustomerFunc(name)
```

Comment:
A gente poderia simplesmente retornar o resultado da funcao wrapped.

```suggestion
		return m.createCustomerFunc(name)
```